pom.xml

```diff
@@ -5,7 +5,7 @@
 	<groupId>org.opentestsystem.shared</groupId>
 	<artifactId>shared-master</artifactId>
 	<name>Master POM for open test system shared modules</name>
-	<version>1.0.1-BUILD-SNAPSHOT</version>
+	<version>3.0.0-SNAPSHOT</version>
```

Comment:
We will be releasing the parent POM and all submodules as a single-version set of artifacts.  (This means that if something changes in 'shared-db,' for example, we will need to re-release the entire set and bump the parent pom version on any child projects.)